modules/ocf/manifests/packages/vscode/apt.pp

```diff
@@ -0,0 +1,18 @@
+# Include VS Code apt repo.
+class ocf::packages::vscode::apt {
+  apt::key { 'vscode':
+    id     => 'BC528686B50D79E339D3721CEB3E94ADBE1229CF',
+    server => 'keyserver.ubuntu.com',
+  }
+
+  apt::source { 'vscode':
+    architecture => 'amd64',
```

Comment:
doesn't actually matter here but you can do:

```puppet
    architecture => $::architecture,
```

Reply:
I think this makes more sense since they only package for amd64 (in case we ever get RISC V desktops lmao)